drivers/hmis/app/graphql/types/hmis_schema/enums/enrollment_status.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::Enums::EnrollmentStatus < Types::BaseEnum
+    description 'Status of this enrollment'
+    graphql_name 'EnrollmentStatus'
+
+    value 'OWN_ENTRY_INCOMPLETE', 'Entry Incomplete'
+    value 'ANY_ENTRY_INCOMPLETE', 'Household Entry Incomplete'
+    value 'ACTIVE', 'Active'
+    value 'EXITED', 'Exited'
+    value 'OWN_EXIT_INCOMPLETE', 'Exit Incomplete'
+    value 'ANY_EXIT_INCOMPLETE', 'Household Exit Incomplete'
+
+    def self.from_enrollment(enrollment, user:)
+      return 'OWN_ENTRY_INCOMPLETE' if enrollment.in_progress?
+
+      return 'OWN_EXIT_INCOMPLETE' if enrollment.exit_date.nil? && enrollment.exit_assessment&.present?
```

Comment:
Can you have an exit_assessment with no ExitDate (maybe that's the WIP version?)

Reply:
Yes. If you have a WIP Exit Assessment, then you don't have an ExitDate.